jorldy/core/env/ml_agent.py

```diff
@@ -111,7 +111,7 @@ def __init__(self, **kwargs):
 
 class DroneMLAgent(_MLAgent):
     def __init__(self, **kwargs):
-        env_name = "drone"
+        env_name = "DroneDelivery"
```

Comment:
other drone files to be changed => drone_delivery
- DroneMLAgent -> DroneDeliveryMLAgent
- config/drone_mlagent -> config/drone_delivery_mlagent